shipstation/api.py

```diff
@@ -20,6 +20,7 @@ def as_dict(self):
             if value is None:
                 d[key] = None
             else:
+                if type(value) == unicode: value = value.encode('ascii', errors='replace')                
```

Comment:
I have a couple of comments:
* This violates PEP8 line length restrictions, please place a new line after the colon
* the `replace` flag will drop weird characters in for every validation error. It may be better to just `ignore`.
** see the examples here: https://docs.python.org/2/howto/unicode.html#the-unicode-type